config/crd/bases/controlplane.cluster.x-k8s.io_rosacontrolplanes.yaml

```diff
@@ -63,6 +63,224 @@ spec:
                   AuditLogRoleARN defines the role that is used to forward audit logs to AWS CloudWatch.
                   If not set, audit log forwarding is disabled.
                 type: string
+              autoNode:
+                description: AutoNode set the auto nde mode and auto node role ARN.
+                properties:
+                  mode:
+                    default: disable
+                    description: AutoNode mode allowed values are enable & disable
+                    enum:
+                    - enable
+                    - disable
+                    type: string
+                  roleARN:
+                    description: |-
+                      AutoNode role ARN that has the IAM Policy and cluster-specific Role that gives permissions to the Karpenter controller.
+                      IAM policy should be as below AND the role must be attached with the same OIDC-ID that is used with the ROSA-HCP cluster.
+                      {
```

Comment:
@serngawy The Policy for the IAM role is highly likely to change. For the private preview we are given this to customers via documentation, so I would suggest not to embed it here.

Reply:
okay